modules/contact/test/tests/bouncing-block-contact/gold/frictional-nodal-min-normal-lm-mortar-fb-tangential-lm-mortar-action_out.e

```diff
@@ -0,0 +1 @@
+frictional-nodal-min-normal-lm-mortar-fb-tangential-lm-mortar-disp_out.e
```

Comment:
It looks like the non-contact test was re-golded. Was the reason for that the renaming of the variables? Did you verify that the results didn't actually change?

Reply:
Yes, I checked the results visually and they looked the same. I suppose there's probably a way to exodiff or extract the raw values and compare the renamed vars to the old ones. Is that necessary here?

Reply:
You can use a custom comparison file like that found [here](https://github.com/idaholab/moose/blob/next/modules/combined/test/tests/sliding_block/in_and_out/constraint/frictionless_kinematic_out.cmp) and you can actually delete the rows corresponding to variables you don't care about. I guess in this case you could delete all but the displacements from the `.cmp` file since those are what we ultimately care about.

However, if they looked the same, then that is probably good enough.

Reply:
If you ever do want to generate a comparison file, you can execute a command like this:
```
$MOOSE_DIR/framework/contrib/exodiff/exodiff -summary exodus_file.e 2>&1 | tee comparison_file_name.cmp
```